README.md

```diff
@@ -1,4 +1,4 @@
-# Astrobee Robot Software
+# Astrobee Robot Software v1
```

Comment:
bad code coverage

---
**Sema Reaction:** :hammer_and_wrench: This code needs a fix

Reply:
Awsome
Always self-close tags that have no children. eslint: react/self-closing-comp

📄 [AirBnB](https://github.com/airbnb/javascript/tree/master/react)

---
**Sema Reaction:** :trophy: This code is awesome | **Sema Tags:** Inefficient, Elegant, Reusable

Reply:
Awesome. But change something
Always self-close tags that have no children. eslint: react/self-closing-comp

📄 [AirBnB](https://github.com/airbnb/javascript/tree/master/react)

---
**Sema Reaction:** :hammer_and_wrench: This code needs a fix | **Sema Tags:** Readable, Not secure